### DISCUSSION TITLE
prioritize interesting files when sorting the file picker

### DISCUSSION THREAD

**Discussion by doy** [helix-editor/helix#7841] [General]
the file picker would be more useful if open buffers and git modified files were sorted at the top, since those are much more likely to be files i am trying to open.

**Comment by pascalkuthe** [comment]
I am not really a fan of messing with the sort order. Firstly this doesn't reallz work when streaming the file system transversal (see #7814) and picker items will always be sorted by fuzzy match score and length first and only in ase of ties (or if there is no pattern) will the sort order even matter so this doesn't have much impact in practice.

I think a better waz to solve this is:
* use the right picker for the job (<space>b for open buffers and git status otherwise)
* use columnar filtering to only show changed/open files (see #7265)
